### ISSUE TITLE
Accessibility A compliance

### ISSUE THREAD

**Issue by vish1988** [Hacker0x01/react-datepicker#1661]
### Expected behavior
All the dates rendered for a month has role=options in it needs an "aria-selected" state attribute to be assigned to it.
### Actual behavior
Currently the aria-selected is missing.
### Steps to reproduce
Open the demo site with Plugins like "siteimprov" to see the issues in compliance.
<img width="1466" alt="Screen Shot 2019-03-11 at 10 38 31" src="https://user-images.githubusercontent.com/7909473/54148220-762ba400-43f9-11e9-95a6-233c24ed23ce.png">

**Comment by J-Son89** [comment]
Also the 'todayButton' should be <button> and not a <div> so it's accessible by keyboard. That or an alternative solution so keyboard navigation works.

**Comment by RomanPerin** [comment]
Hi guys!
It is a pity that it was closed without addressing...(((( 
There might not be too much effort to add `aria-selected` according to the specification: https://www.w3.org/TR/wai-aria-practices/examples/dialog-modal/datepicker-dialog.html.
I would very appreciate for giving the hope of adding it in upcoming updates)

**Comment by martijnrusschen** [comment]
@RomanPerin Feel free to create a PR to improve this.
